src/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $day-size;
+    width: $day-size;
+    border: 1px solid #000;
+    background-color: $day-color;
+    margin-right: 1px;
+    cursor: pointer;
+    transition: ease 500ms;
+
+    &:hover {
+      background-color: $hover-color;
+      transform: translateY(-20px);
+    }
+
+    @for $item from 1 through 31 {
+      &:nth-child(#{$item}):before {
+        content: "#{$item}";
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 30px;
```

Comment:
Don't use SASS loops for styles that stay the same for all elements of the group, e.g. display or position.